extensions/cornerstone/src/init.js

```diff
@@ -220,45 +224,45 @@ export default function init({ servicesManager, configuration }) {
   csTools.setToolEnabled('Overlay', {});
 }
 
-const _initMeasurementService = measurementService => {
+const _initMeasurementService = (MeasurementService, DisplaySetService) => {
```

Comment:
Only weird bit is that this doesn't have it's own lexical scope. Does it conflict with the `MeasurementService` we're pulling out of the `servicesManager.services`?

Reply:
It doesn't conflict, this is a private function outside of the scope of the main init call.